public/scripts/gameHandler.js

```diff
@@ -33,6 +43,44 @@ function displayRole(role) {
   description.textContent = role.description
 }
 
+function showIsActivePlayer(activePlayer) {
+  const playerListArray = Array.from(playerList.childNodes)
+  const players = playerListArray.filter(function(node) {
+    return node.nodeName === 'LI'
+  })
+  for (let player of players) {
+    if (player.innerText === activePlayer) {
+      player.innerText = '> ' + activePlayer
+    } else if (player.innerText.substring(0, 2) === '> ') {
+      player.innerText = player.innerText.substring(2)
+    }
+  }
+}
+
+function progressGame(newJob, roundCount, activePlayer) {
+  roundCounter.textContent = `ROUND ${roundCount}`
+  showIsActivePlayer(activePlayer)
+  if (newJob === 'chat') {
+    wordCounter.hidden = false
+    customInput.hidden = false
+    requiredWordsList.hidden = false
+    alert('your turn!')
+  } else if (newJob === 'wait') {
+    wordCounter.hidden = false
+    customInput.hidden = false
+    requiredWordsList.hidden = false
+    alert(activePlayer + "'s turn")
+  } else if (newJob === 'vote') {
+    alert('vote!')
+  } else if (newJob === 'done') {
+    wordCounter.hidden = true
+    customInput.hidden = true
+    requiredWordsList.hidden = true
+  } else if (newJob === 'end') {
+    alert('THE END!')
+  }
```

Comment:
can you make this whole thing a switch statement rather than if else if else if else if?